locales/en/breach-checklist.ftl

```diff
@@ -0,0 +1,72 @@
+# Strings for the breach details checklists
+
+## Prompts the user for changes when there is a breach detected of password
+
+# NOTE: { $companyWebsite } is a placeholder for the URL to the website of the company where the breach occurred 
+breach-checklist-pw-header = Go to <a>{$companyWebsite}</a> to change your password and enable two-factor authentication (2FA). 
+breach-checklist-pw-body = Make sure your password is unique and hard to guess. If this password is used on any other accounts, be sure to change it there too. <a>{-brand-name} Password Manager</a> can help you securely keep track of all of your passwords.
```

Comment:
We also have some uses of `{-brand-name}` in `app.ftl` and `bento.ftl`.. Would we want to also change those to `-brand-firefox` for consistency?

Reply:
Did it anyways

Reply:
Sorry I didn't reply in time; looks like @flodolo caught this though.  We don't want/need to change the old stuff, it will be deprecated.  Until then, it's fine for the old site to refer to the old strings.  

We will create new .ftl files as needed for the new site, and things that are site-wide can go at the bottom of `app.ftl` with the other new strings.  After old site is retired, we'll delete the old strings from `app.ftl`.